searx/engines/openstreetmap.py

```diff
@@ -349,19 +350,23 @@ def get_links(result, user_language):
     links = []
     link_keys = set()
     for k, mapping_function in VALUE_TO_LINK.items():
-        raw_value = result['extratags'].get(k)
-        if raw_value:
-            url, url_label = mapping_function(raw_value)
-            if url.startswith('https://wikidata.org'):
-                url_label = result.get('wikidata', {}).get('itemLabel') or url_label
-            links.append(
-                {
-                    'label': get_key_label(k, user_language),
-                    'url': url,
-                    'url_label': url_label,
-                }
-            )
-            link_keys.add(k)
+        extratags = result.get('extratags')
+        if not isinstance(extratags, dict):
+            continue
```

Comment:
Actually this condition should be before the loop : no need to iterate over VALUE_TO_LINK